step/step.go

```diff
@@ -21,7 +21,7 @@ const (
 	// - `**/gradle-wrapper.properties`: contains exact Gradle version
 	// - `**/gradle.properties`: contains Gradle config values
 	// - `**/gradle/libs.versions.toml`: version catalog file, contains dependencies and their versions
```

Comment:
NIT: As far as I see, all the elements of the key have some explanation comments, I suggest adding one for the new init script-related elements too.